### PR TITLE
chore(package): update Babel 7

### PR DESCRIPTION
**Pull request by layershifter**
# Babel 7 Released

https://babeljs.io/blog/2018/08/27/7.0.0

---

It's time to go away from beta :+1:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
   "author": "Felix Kling",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "@babel/parser": "7.0.0-beta.53",
+    "@babel/parser": "7.0.0",
```
Comment (danez) [inline package.json]: can you now please change this to `^7.0.0`

Comment (layershifter) [reply package.json]: Oops, will do!

**Comment by jquense** [comment]
I've been working on updating `gatsby-transformer-react-docgen` to babel 7 by implementing our own parser over recdocgen, but it seems like bable7 somewhere changes `Property` to `ObjectProperty` for property types, which breaks the proptypes handler...i'm a bit confused why its not breaking here tho...

**Comment by pksunkara** [comment]
@danez Can we please get this merged and a new release be done? This kinda blocks the whole toolchain from being used because of licensing issues in babel beta.

File: package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
   "author": "Felix Kling",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "@babel/parser": "7.0.0-beta.53",
+    "@babel/parser": "^7.0.0",
```
Comment (pksunkara) [inline package.json]: @layershifter You can do `^7.1.0` now

File: package.json [hunk 3]
```diff
@@ -36,7 +36,7 @@
...
     "async": "^2.1.4",
     "babel-runtime": "^6.9.2",
```
Comment (pksunkara) [inline package.json]: Let's use `@babel/runtime` and change it to v7.

**Comment by danez** [comment]
Thank you

**Pull request merged**